packages/web-frontend/src/selectors/projectSelectors.js

```diff
@@ -37,7 +37,9 @@ export const selectTileSets = createSelector(selectCurrentProject, project => {
 
   if (project.config && project.config.tileSets) {
     const customSetKeys = project.config.tileSets.split(',').map(item => item.trim());
-    tileSetKeys = [...tileSetKeys, ...customSetKeys];
+    tileSetKeys = customSetKeys.includes('laosOpenStreets', 'laosSatellite')
```

Comment:
I can see the intent here, to have laos_eoc only have the laos tilesets. But, this is putting laos specific logic into a function that should work for all projects equally.

My suggestion is to instead add a property to the project config, `includeDefaultTilesets` (default true). This can be set to false for laos_eoc.

The code then becomes: `tileSetKeys = includeDefaultTilesets ? [...tileSetKeys, ...customSetKeys] : customSetKeys`.

The defaults `osm, satellite` are defined above on line 36.

Reply:
Thanks Igor, this looks great!
Additionally I suggest we use **not**IncludeDefaultTilesets, if not, in other projects `includeDefaultTilesets` is missing and false, and that will not include default tile sets.

Reply:
Ah yep, I was thinking if it was `includeDefaultTilesets` you can leave it out of the others, and then in this code here you can give it a default of true e.g.

```
const { includeDefaultTilesets = true } = project.config;
```

Both ways make sense, I just like to use positive variable names as they are usually easier for me to read and understand quickly.

Reply:
That's a smart way to play with code. I agree using positive variable name is better. Will implement this suggestion.